lib/order.rb

```diff
@@ -0,0 +1,98 @@
+class Order
+  attr_accessor :products, :customer, :fulfillment_status
+  attr_reader :id
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    # fulfillment_status = :pending <-- this is the syntax to set a default value for a positional argument
+
+    valid_fulfillment_statuses = [:pending, :paid, :processing, :shipped, :complete]
+    if !valid_fulfillment_statuses.include?(fulfillment_status)
```

Comment:
Nice logic! It might make sense to push this further and even pull this to a constant variable!